VS-UtCS-Manual/Makefile.common

```diff
@@ -1,52 +1,36 @@
 ## build rules
 
-TEX_DEPS = overview.tex timeline.tex PCplots.tex random.tex \
+TEX_DEPS = overview.tex \
 	appendixA.tex appendixB.tex appendixC.tex \
 	splashnotice.tex remark.tex prettygloss.tex \
-	nomencldef.tex preface.tex texheader.tex \
-	profiles.tex glossarydefs.tex VSUreality.tex lifeinvsu.tex \
-	astrography-is-destiny.tex \
-	portfolio_Aeran.tex portfolio_Lmpl.tex \
-	portfolio_AndolianProtectorate.tex portfolio_Luddites.tex \
-	portfolio_Andolian.tex portfolio_Mechanist.tex \
-	portfolio_Bzbr.tex portfolio_Mishtali.tex \
-	portfolio_Confederation.tex portfolio_Nuhln.tex \
-	portfolio_CriminalElements.tex portfolio_Purist.tex \
-	portfolio_Dgn.tex portfolio_Purth.tex \
-	portfolio_Forsaken.tex portfolio_RlaanBriin.tex \
-	portfolio_Highborn.tex portfolio_Rlaan.tex \
-	portfolio_HomelandSecurity.tex portfolio_Saahasayaay.tex \
-	portfolio_HuntersGuild.tex portfolio_Shaper.tex \
-	portfolio_ISMG.tex portfolio_Shmrn.tex \
-	portfolio_ISO.tex \
-	portfolio_Klk'k.tex portfolio_Uln.tex \
-	portfolio_LIHW.tex portfolio_Unadorned.tex
```

Comment:
why dropping to many files? Or are the specfieid a different way?

Reply:
They were deleted by @ministerofinformation in https://github.com/vegastrike/User-Documentation/commit/3755e6e4a07129a1130e20ffe6be3ba16323d0eb, as he restructured the TeX code, but I guess he forgot to adapt the build system.